### PR TITLE
chore: update AFGO

### PR DESCRIPTION
**Pull request by aholovko**
Signed-off-by: Andriy Holovko <andriy.holovko@gmail.com>

### CODE REVIEW

File: k8s/jobs/kustomize/jobs/overlays/common/aries-framework/jsonld/add_contexts.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+apk --no-cache add curl
+
+ENDPOINTS=(adapter-issuer adapter-rp demo-issuer demo-rp governance-vcs holder-vcs issuer-vcs verifier-vcs);
```
Comment (alialkhalidi) [inline k8s/jobs/kustomize/jobs/overlays/common/aries-framework/jsonld/add_contexts.sh:10]: change this to:
`ENDPOINTS="adapter-issuer adapter-rp demo-issuer demo-rp governance-vcs holder-vcs issuer-vcs verifier-vcs"`

File: k8s/jobs/kustomize/jobs/overlays/common/aries-framework/jsonld/add_contexts.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+for endpoint in "${ENDPOINTS[@]}";
```
Comment (alialkhalidi) [inline k8s/jobs/kustomize/jobs/overlays/common/aries-framework/jsonld/add_contexts.sh:11]: change this to:
`for endpoint in ${ENDPOINTS};

File: k8s/jobs/kustomize/jobs/overlays/common/aries-framework/jsonld/add-contexts.yml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            - "-c"
+            - |
+              ENDPOINTS=(demo-issuer demo-rp);
+              for endpoint in "${ENDPOINTS[@]}";
+              do while [[ "$(curl -o /dev/null -s -w '%{http_code}' --insecure --connect-timeout 5 https://${endpoint}.||DOMAIN||/healthcheck)" != "200" ]];
+              do echo "waiting for ${endpoint} endpoint";
+              sleep 5;
+              done;
+              done;
+        - name: vcs-components-ready
```
Comment (alialkhalidi) [inline k8s/jobs/kustomize/jobs/overlays/common/aries-framework/jsonld/add-contexts.yml:52]: might want to move this to become the second entry before the demo-issuer/rp

Comment (aholovko) [reply k8s/jobs/kustomize/jobs/overlays/common/aries-framework/jsonld/add-contexts.yml:38]: removed demo-issuer/rp as no need to add contexts for them thru API

**Review by fqutishat — APPROVED** [review]

**Pull request merged**